### PR TITLE
[WIP] Update documentation

### PR DESCRIPTION
**Pull request by vagkout**
### Useful Links
[CrowdHeritage-AI4Culture change list](https://docs.google.com/document/d/1zSJByc7Okn78bOk4z4UJeY8K4QQBCJxjPVAy_7YLgCA/edit#heading=h.ahx8543cb1n6)
[Image Comparison doc (used for the specific section doc)](https://docs.google.com/document/d/1IXH2I27FlY5U0QIz_5ImdytMkiJCX6_5YzhsbPD5P94/edit)

### How and Why
As part of documenting the tool's change list for AI4Culture we needed two things:
- document all the new changes that were part of AI4Culture
- update stale docs (stale UI, features added in the past and never mentioned)

**Note** : 
I wanted to focus on the core stuff and believe the current changes are enough for now. 
Two things can be improved:
- The campaign annotation section could be described in more detail
- Although I did update some stale UI sections I did leave some with the old tool bar (minor UI changes-do not really affect docs) since I would need to update all the screenshots because the campaign used is not active.

### How It Looks Like
Since there are updates that touch several points of the documentation there is not a meaningful artefact I can provide for this section.

The suggested way of reviewing:
1. First check the diff to review the texts that changed (I left comments on parts I was not sure about)
2. Open the markdown locally to see how it looks like and do a quick pass through all the documentation 

### How to Test It
Can check it locally by using `Visual Studio Code` or another editor that supports markdown preview.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -56,10 +63,31 @@ The second annotation type is a **comment**. Comments are text annotations, just
...
 
 ### 3. GEOTAGGING
-Lastly, **geotagging** is a special kind of tag that refers specifically to a location. A user can start typing the location name in the input field under the map, and then select the appropriate location from the suggested ones on the drop-down below. Geotags can also be *Upvoted*, *Downvoted* or *Removed* by other users.
+**Geotagging** is a special kind of tag that refers specifically to a location. A user can start typing the location name in the input field under the map, and then select the appropriate location from the suggested ones on the drop-down below. Geotags can also be validated by the supported feedback methods (upvoting or percentage rating).
 
 <img src="https://raw.githubusercontent.com/ails-lab/CrowdHeritage_Documentation/main/docs/_media/image9.jpg" width="auto">
 
+### 4. COLORTAGGING
+
+**Colortagging** is a special kind of tag that refers to color. A user can select the color from a predefined color pallette. Colortags can also be validated by the supported validation methods (upvoting or percentage rating).
```
Comment (vagkout) [inline docs/README.md]: Verify this

File: docs/README.md [hunk 2]
```diff
@@ -56,10 +63,31 @@ The second annotation type is a **comment**. Comments are text annotations, just
...
+
+<img src="_media/annotating-colortagging.png" width="auto">
+
+### 5. IMAGETAGGING
+
+**IMAGETAGGING** is a special kind of tag that refers to tagging images. A user can see the original image along with algorithm-generated ones (e.g. by Super-Resolution models). The user can then click *Compare* to start the comparison.
```
Comment (vagkout) [inline docs/README.md]: Verify this. 

BTW Is it only Super-Resolution models?

File: docs/README.md [hunk 3]
```diff
@@ -56,10 +63,31 @@ The second annotation type is a **comment**. Comments are text annotations, just
...
+
+<img src="_media/annotating-image-tagging.png" width="auto">
+
+The original image is displayed at the top of the screen and below it (from left to right) the first algorithm-generated image, the original image in the same size as the computer-generated ones and the second algorithm-generated image. The user can zoom in a smaller window of the original image at the top and automatically the images at the bottom are displaying this smaller window. If the user wants to see the full images again he can stop the zoom process.
```
Comment (vagkout) [inline docs/README.md:81]: The rest regarding image comparison were copied from the respective doc. 

https://docs.google.com/document/d/1IXH2I27FlY5U0QIz_5ImdytMkiJCX6_5YzhsbPD5P94/edit

File: docs/README.md [hunk 4]
```diff
@@ -23,9 +23,13 @@ Each Campaign consists of one or more Collections. You can view those Collection
 <img src="https://raw.githubusercontent.com/ails-lab/CrowdHeritage_Documentation/main/docs/_media/image3.jpg" width="auto">
 
 ## Selecting an Item
-Within the Collection, you will be presented with a description of that Collection and the Items it contains. You also have the option to view or hide the Items you have already annotated, by selecting the ***View*** or ***Hide*** option on the selector.
+Within the Collection, you will be presented with a description of that Collection and the Items it contains. You can choose which items you want to see, by selecting one of the *ALL ITEMS*, *MY ANNOTATED ITEMS*, *NOT ANNOTATED ITEMS* options on the selector.
 
-<img src="https://raw.githubusercontent.com/ails-lab/CrowdHeritage_Documentation/main/docs/_media/image4.jpg" width="auto">
+<img src="_media/collection-editor-view-options.png" width="auto">
```
Comment (vagkout) [inline docs/README.md:28]: Newly recorded images I added them with a relative path so that we can preview them locally.

I expect this to work for deployment as well, as with normal websites but will double check upon deployment. 

Had trouble installing Docsify locally and did not want to spend time.

File: docs/README.md [hunk 5]
```diff
@@ -121,27 +180,35 @@ When a Campaign Owner visits a Campaign page, a button named "Moderate Campaign"
...
-The final two buttons give the Campaign Owner the option to export the contributing users and the annotations of the Campaign, as a JSON file.
+### Data Export
+By clicking the *Data Export* button, the Campaign Owner can see the two campaign export options, *EXPORT CONTRIBUTORS* and *EXPORT ANNOTATIONS*.
+
+These two buttons allow the Campaign Owner to export the contributing users and the annotations of the Campaign, as a JSON file.
+
+<img src="_media/campaign-moderate-data-export.png" width="auto">
+
+#### Export Annotations for MINT
+The MINT tool supports controlled import of the annotations exported from CrowdHeritage.
```
Comment (vagkout) [inline docs/README.md]: **Pending**

Until we know exactly how it works and have the screenshots as well.

File: docs/README.md [hunk 6]
```diff
@@ -35,9 +39,12 @@ If you want to create annotations for am Item, click on it to access its page.
 
 ## Annotating an Item
 There are three annotation types one can add to an Item.
```
Comment (vagkout) [inline docs/README.md]: Typo  -> five

File: docs/README.md [hunk 7]
```diff
@@ -56,10 +63,31 @@ The second annotation type is a **comment**. Comments are text annotations, just
 Comments can also be *Upvoted*, *Downvoted* or *Removed* by other users. 
 
 ### 3. GEOTAGGING
-Lastly, **geotagging** is a special kind of tag that refers specifically to a location. A user can start typing the location name in the input field under the map, and then select the appropriate location from the suggested ones on the drop-down below. Geotags can also be *Upvoted*, *Downvoted* or *Removed* by other users.
+**Geotagging** is a special kind of tag that refers specifically to a location. A user can start typing the location name in the input field under the map, and then select the appropriate location from the suggested ones on the drop-down below. Geotags can also be validated by the supported feedback methods (upvoting or percentage rating).
```
Comment (vagkout) [inline docs/README.md]: This can only be voted - revert this change

File: docs/README.md (hunk 1 above)
Comment (vagkout) [reply docs/README.md]: Revert to voting method

File: docs/README.md (hunk 2 above)
Comment (vagkout) [reply docs/README.md]: Spiros ->
Image tagging only works with base annotations

Suggestion ->
"refers to tagging images by making use of the base annotation functionality"

File: docs/README.md (hunk 5 above)
Comment (vagkout) [reply docs/README.md]: Update based on what we discussed with Spiros and add screenshot

File: docs/README.md [hunk 8]
```diff
@@ -34,13 +38,15 @@ By hovering on the thumbnail of an Item, you can either click on it to enter the
 If you want to create annotations for am Item, click on it to access its page.
 
 ## Annotating an Item
```
Comment (spyrosbek) [inline docs/README.md:40]: Since we have a same-level header called "Image Comparison", maybe we could rename this header into something similar, like "Basic Campaign - Annotating an Item"?

File: docs/README.md [hunk 9]
```diff
@@ -49,17 +55,38 @@ There is also the option to add a new tag by typing it in the input field. Since
...
 
 <img src="https://raw.githubusercontent.com/ails-lab/CrowdHeritage_Documentation/main/docs/_media/image9.jpg" width="auto">
 
+### 4. COLORTAGGING
+
+**Colortagging** is a special kind of tag that refers to color. A user can select the color from a predefined color pallette. Colortags can also be *Upvoted*, *Downvoted* or *Removed* by other users.
+
+<img src="_media/annotating-colortagging.png" width="auto">
+
+## Image Comparison
```
Comment (spyrosbek) [inline docs/README.md:75]: Similarly with the previous comment, maybe something like "Image Comparison Campaign - Ranking a set of images"?

File: docs/README.md [hunk 10]
```diff
@@ -70,17 +97,76 @@ To access the dashboard, a signed-in Campaign Organizer should click on his user
...
+This is a long list of input fields like: Visibility, Start/End Dates, Input Language, Banner, Description and so on. There are buttons to *Preview* and/or *Save* the Campaign after you are done.
 
 <img src="https://raw.githubusercontent.com/ails-lab/CrowdHeritage_Documentation/main/docs/_media/image13.jpg" width="auto">
 
 The input fields marked as *(multilingual)* can be edited in multiple languages by selecting their appropriate language on the **Input Language** dropdown menu on top. This way, whenever a user views a Campaign, it will appear in their own language if the Campaign Owner has added the relevant text in the user's language.
 
+#### Campaign Annotating Section
+
+First, the user needs to select one of the available campaign types (basic, translate, image comparison) through the *campaign type* option.
+Based on the campaign type different options become availablem.
```
Comment (spyrosbek) [inline docs/README.md:120]: Proposed changes:
Based on the **selected** campaign type different **campaign parameters** become **available** (typo).

File: docs/README.md [hunk 11]
```diff
@@ -70,17 +97,76 @@ To access the dashboard, a signed-in Campaign Organizer should click on his user
...
+
+<img src="_media/campaign-editor-campaign-annotating-types.png" width="auto">
+
+##### Campaign Type: Basic
+
+The basic campaign type caters for the majority of scenarios a Campaign Organiser needs. It supports creating simple campaigns that are concerned with a single annotation type (e.g. semantic tagging), but it also supports the creation of complex campaigns, where the campaign organisers can specify multiple annotation types (semantic tagging,  colortagging, geotagging, commenting). The desired annotation types are specified via the “Campaign motivations'' multi-select option.
```
Comment (spyrosbek) [inline docs/README.md]: Typo:
“Campaign motivations'' (different end quotes)

File: docs/README.md [hunk 12]
```diff
@@ -119,30 +207,38 @@ When a Campaign Owner visits a Campaign page, a button named "Moderate Campaign"
...
+### Statistics
+By clicking the *Statistics* button, the Campaign Owner can view some statistics about that specific Campaign, like total items, total annotations, annotation upvotes, contribution status  etc.
 
-<img src="https://raw.githubusercontent.com/ails-lab/CrowdHeritage_Documentation/main/docs/_media/image21.jpg" width="auto">
+<img src="_media/campaign-moderate-statistics.png" width="auto">
 
-### View annotations by TAG
-By typing or selecting one of the available tags, like "Bass saxophone", a list of Items annotated with the chosen tag will appear.
+### Validation
+By clicking the *Validation* button the Campaign Owner can validate the campaign annotations. By typing or selecting one of the available tags, like "Bass saxophone", a list of Items annotated with the chosen tag will appear.
```
Comment (spyrosbek) [inline docs/README.md]: These 2 sentences are irrelevant since we have hidden the "Campaign Bulk Validation" section (for now). Since we have only kept the "Campaign Export Criteria" section, we could change these sentences to something like:

By clicking on the "Validation" tab, the Campaign Owner can automatically validate the produced annotations, based on the feedback (upvotes/downvotes) they have received by the campaign contributors.

File: docs/README.md [hunk 13]
```diff
@@ -119,30 +207,38 @@ When a Campaign Owner visits a Campaign page, a button named "Moderate Campaign"
...
+By clicking the *Data Export* button, the Campaign Owner can see the two campaign export options, *EXPORT CONTRIBUTORS* and *EXPORT ANNOTATIONS*.
 
-### Publish Criteria
-The *Publish Criteria* button opens a pop-up page with **FIXME: Not sure what that is?**
+#### EXPORT ANNOTATIONS
+The Campaign Owner is presented with two options for exporting annotations. They can either copy the annotations download link into the clipboard or directly download the annotations file.
 
-<img src="https://raw.githubusercontent.com/ails-lab/CrowdHeritage_Documentation/main/docs/_media/image24.jpg" width="auto">
+#### EXPORT Contibutors
+There are two ways of exporting the campaign contributors: as CSV file (which contains a table with some basic information regarding the users), or as JSON file (which contains more detailed information about the contributing users)
```
Comment (spyrosbek) [inline docs/README.md]: Add full stop at the end of the sentence.

**Review by spyrosbek — CHANGES_REQUESTED** [review]
Very good!
I've left just some small remarks as comments on the changed files.
